### PR TITLE
Simplify generation of `io_handler.get_output_directory()`.

### PR DESCRIPTION
**Pull request by GernotMaier**
This is a maintenance PR triggered by issue #760 .

The function `io_handler.get_output_directory` is used frequently throughout the code. The usage is slightly inconsistent, as only `use_plain_path` together with `dir_type=simtools-result` returns exactly the `output_path` as set through the command line configuration. This is not a naively expected behaviour.

This PR simplifies the generation of `io_handler.get_output_directory()`:

- removal of `dir_type` option. This was hardly used in the code (mostly with the `test` option to add 'test' to the output directory; once with `simtools-results`)
- `use_plain_path`  is now always returning exactly the `output_path` as set through the command line configuration (anything else can be steered through the `sub_dir` option).
- refactored the `io_handler.get_output_directory()` (removed some if statements by changing the logic) and completed the unit tests

For reviewing:

- relevant code changes in io_handler.py and test_io_handler.py only 
- all other code changes are the removal of `dir_type`
- all other changes are done by  using `sed` for unit and integration tests to adjust to the new behavior.
- note that small change in retrieving the model directory in simtools/simtel/simulator_light_emission.py ([link](https://github.com/gammasim/simtools/blob/7d9f036ce4735b15e06f1e92d1d5dcde2cb78184/simtools/simtel/simulator_light_emission.py#L229)

### CODE REVIEW

**Comment by tobiaskleiner** [comment]
Running the integration tests locally gives one failed test:


`run000010_gamma_za20deg_azm000deg_South_test_layout_test-production-South-pack-for-grid.log_hist` gets stored in the base simtools directory, which leads to the following error:

```

================================================ FAILURES =================================================
_________ test_applications_from_config[simtools-simulate-prod_gamma_20_deg_pack_for_south_grid] __________
[gw2] linux -- Python 3.11.7 /workdir/env/bin/python3.11

tmp_test_directory = local('/tmp/pytest-of-root/pytest-0/popen-gw2/test-data10')
config = {'APPLICATION': 'simtools-simulate-prod', 'CONFIGURATION': {'ARRAY_LAYOUT_NAME': 'test_layout', 'AZIMUTH_ANGLE': 'Nort...zm000deg_South_test_layout_test-production-South-pack-for-grid.zst'}], 'TEST_NAME': 'gamma_20_deg_pack_for_south_grid'}
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0xffff6276fa10>
request = <FixtureRequest for <Function test_applications_from_config[simtools-simulate-prod_gamma_20_deg_pack_for_south_grid]>>
model_version = '6.0.0'

    @pytest.mark.parametrize(
        "config",
        get_list_of_test_configurations(),
        ids=get_list_of_test_configurations(get_test_names=True),
    )
    def test_applications_from_config(tmp_test_directory, config, monkeypatch, request, model_version):
        """
        Test all applications from config files found in the config directory.
    
        Parameters
        ----------
        tmp_test_directory: str
            Temporary directory, into which test configuration and output is written.
        config: dict
            Dictionary with the configuration parameters for the test.
    
        """
    
        # The db_add_file_to_db.py application requires a user confirmation.
        # With this line we mock the user confirmation to be y for the test
        # Notice this is done for all tests, so keep in mind if in the future we add tests with input.
        monkeypatch.setattr("sys.stdin", StringIO("y\n"))
    
        try:
            tmp_output_path = Path(tmp_test_directory).joinpath(
                config["APPLICATION"] + "-" + config["TEST_NAME"]
            )
        except KeyError as exc:
            logger.error(f"No application defined in config file {config}.")
            raise exc
        tmp_output_path.mkdir(parents=True, exist_ok=True)
        logger.info(f"Temporary output path: {tmp_output_path}")
        logger.info(f"Test configuration from config file: {config}")
        logger.info(f"Model version: {request.config.getoption('--model_version')}")
        if "CONFIGURATION" in config:
            model_version_requested = request.config.getoption("--model_version")
            if "MODEL_VERSION_USE_CURRENT" in config:
                model_version_config = config["CONFIGURATION"]["MODEL_VERSION"]
                if model_version_requested != model_version_config:
                    pytest.skip(
                        "Model version requested {model_version_requested} not supported for this test"
                    )
            config_file, config_string, config_file_model_version = prepare_configuration(
                config["CONFIGURATION"],
                output_path=tmp_output_path,
                model_version=model_version_requested,
            )
        else:
            config_file = None
            config_string = None
            config_file_model_version = None
    
        cmd = get_application_command(
            app=config.get("APPLICATION", None),
            config_file=config_file,
            config_string=config_string,
        )
        logger.info(f"Application configuration: {config}")
    
        logger.info(f"Running application: {cmd}")
>       assert os.system(cmd) == 0
E       AssertionError: assert 256 == 0
E        +  where 256 = <built-in function system>('simtools-simulate-prod --config /tmp/pytest-of-root/pytest-0/popen-gw2/test-data10/simtools-simulate-prod-gamma_20_deg_pack_for_south_grid/tmp_config.yml')
E        +    where <built-in function system> = os.system

tests/integration_tests/test_applications_from_config.py:379: AssertionError
------------------------------------------ Captured stderr call -------------------------------------------
INFO::test_applications_from_config(l350)::test_applications_from_config::Temporary output path: /tmp/pytest-of-root/pytest-0/popen-gw2/test-data10/simtools-simulate-prod-gamma_20_deg_pack_for_south_grid
INFO::test_applications_from_config(l351)::test_applications_from_config::Test configuration from config file: {'APPLICATION': 'simtools-simulate-prod', 'TEST_NAME': 'gamma_20_deg_pack_for_south_grid', 'CONFIGURATION': {'SIMULATION_SOFTWARE': 'corsika_simtel', 'LABEL': 'test-production-South-pack-for-grid', 'MODEL_VERSION': '6.0.0', 'SITE': 'South', 'ARRAY_LAYOUT_NAME': 'test_layout', 'PRIMARY': 'gamma', 'RUN_NUMBER_START': 10, 'NUMBER_OF_RUNS': 1, 'AZIMUTH_ANGLE': 'North', 'ZENITH_ANGLE': 20, 'NSHOW': 5, 'ESLOPE': -2.0, 'VIEW_CONE': '0 deg 0 deg', 'ENERGY_RANGE': '100 GeV 1 TeV', 'CORE_SCATTER': '10 1400 m', 'DATA_DIRECTORY': 'simtools-output', 'OUTPUT_PATH': 'simtools-output', 'PACK_FOR_GRID_REGISTER': True}, 'INTEGRATION_TESTS': [{'OUTPUT_FILE': 'directory_for_grid_upload/run000010_gamma_za20deg_azm000deg_South_test_layout_test-production-South-pack-for-grid.log_hist.tar.gz'}, {'OUTPUT_FILE': 'directory_for_grid_upload/run000010_gamma_za20deg_azm000deg_South_test_layout_test-production-South-pack-for-grid.zst'}]}
INFO::test_applications_from_config(l352)::test_applications_from_config::Model version: None
INFO::test_applications_from_config(l312)::prepare_configuration::Writing config file: /tmp/pytest-of-root/pytest-0/popen-gw2/test-data10/simtools-simulate-prod-gamma_20_deg_pack_for_south_grid/tmp_config.yml
INFO::test_applications_from_config(l376)::test_applications_from_config::Application configuration: {'APPLICATION': 'simtools-simulate-prod', 'TEST_NAME': 'gamma_20_deg_pack_for_south_grid', 'CONFIGURATION': {'SIMULATION_SOFTWARE': 'corsika_simtel', 'LABEL': 'test-production-South-pack-for-grid', 'MODEL_VERSION': '6.0.0', 'SITE': 'South', 'ARRAY_LAYOUT_NAME': 'test_layout', 'PRIMARY': 'gamma', 'RUN_NUMBER_START': 10, 'NUMBER_OF_RUNS': 1, 'AZIMUTH_ANGLE': 'North', 'ZENITH_ANGLE': 20, 'NSHOW': 5, 'ESLOPE': -2.0, 'VIEW_CONE': '0 deg 0 deg', 'ENERGY_RANGE': '100 GeV 1 TeV', 'CORE_SCATTER': '10 1400 m', 'DATA_DIRECTORY': '/tmp/pytest-of-root/pytest-0/popen-gw2/test-data10/simtools-simulate-prod-gamma_20_deg_pack_for_south_grid/simtools-output', 'OUTPUT_PATH': '/tmp/pytest-of-root/pytest-0/popen-gw2/test-data10/simtools-simulate-prod-gamma_20_deg_pack_for_south_grid/simtools-output', 'PACK_FOR_GRID_REGISTER': True, 'USE_PLAIN_OUTPUT_PATH': True}, 'INTEGRATION_TESTS': [{'OUTPUT_FILE': 'directory_for_grid_upload/run000010_gamma_za20deg_azm000deg_South_test_layout_test-production-South-pack-for-grid.log_hist.tar.gz'}, {'OUTPUT_FILE': 'directory_for_grid_upload/run000010_gamma_za20deg_azm000deg_South_test_layout_test-production-South-pack-for-grid.zst'}]}
INFO::test_applications_from_config(l378)::test_applications_from_config::Running application: simtools-simulate-prod --config /tmp/pytest-of-root/pytest-0/popen-gw2/test-data10/simtools-simulate-prod-gamma_20_deg_pack_for_south_grid/tmp_config.yml
INFO::simulator(l198)::_validate_run_list_and_range::run_list: [10]
INFO::db_handler(l147)::_find_latest_simulation_model_db::Updated the DB simulation model to the latest version CTAO-Simulation-Model-v0-6-0
INFO::simulator(l262)::simulate::Submission command: local
INFO::simulator(l267)::simulate::Starting submission for 1 run
INFO::array_model(l206)::export_simtel_array_config_file::Writing array configuration file into /tmp/pytest-of-root/pytest-0/popen-gw2/test-data10/simtools-simulate-prod-gamma_20_deg_pack_for_south_grid/simtools-output/CTA-test_layout-South-6.0.0_test-production-South-pack-for-grid.cfg
INFO::corsika_simtel_runner(l111)::_export_multipipe_script::Multipipe script: /tmp/pytest-of-root/pytest-0/popen-gw2/test-data10/simtools-simulate-prod-gamma_20_deg_pack_for_south_grid/simtools-output/multi_cta-South-test_layout.cfg
INFO::corsika_config(l409)::generate_corsika_input_file::Exporting CORSIKA input file to /tmp/pytest-of-root/pytest-0/popen-gw2/test-data10/simtools-simulate-prod-gamma_20_deg_pack_for_south_grid/simtools-output/corsika_config_gamma_South_test_layout_za020-azm000deg_test-production-South-pack-for-grid.input
INFO::job_manager(l107)::submit::Submitting script /tmp/pytest-of-root/pytest-0/popen-gw2/test-data10/simtools-simulate-prod-gamma_20_deg_pack_for_south_grid/simtools-output/sub_scripts/sub_run000010_gamma_za20deg_azm000deg_South_test_layout_test-production-South-pack-for-grid.sh
INFO::job_manager(l108)::submit::Job output stream /tmp/pytest-of-root/pytest-0/popen-gw2/test-data10/simtools-simulate-prod-gamma_20_deg_pack_for_south_grid/simtools-output/sub_logs/sub_run000010_gamma_za20deg_azm000deg_South_test_layout_test-production-South-pack-for-grid.out
INFO::job_manager(l109)::submit::Job error stream /tmp/pytest-of-root/pytest-0/popen-gw2/test-data10/simtools-simulate-prod-gamma_20_deg_pack_for_south_grid/simtools-output/sub_logs/sub_run000010_gamma_za20deg_azm000deg_South_test_layout_test-production-South-pack-for-grid.err
INFO::job_manager(l110)::submit::Job log stream /tmp/pytest-of-root/pytest-0/popen-gw2/test-data10/simtools-simulate-prod-gamma_20_deg_pack_for_south_grid/simtools-output/sub_logs/sub_run000010_gamma_za20deg_azm000deg_South_test_layout_test-production-South-pack-for-grid.job
INFO::job_manager(l129)::_submit_local::Running script locally
INFO::simulate_prod(l177)::main::Production run is complete for primary gamma showers coming from 0.0 deg azimuth and zenith angle of 20.0 deg at the South site, using the 6.0.0 simulation model.
INFO::simulate_prod(l141)::pack_for_register::Packing the output files for registering on the grid
INFO::simulator(l444)::get_file_list::Getting list of output files
INFO::simulator(l444)::get_file_list::Getting list of log files
INFO::simulator(l444)::get_file_list::Getting list of hist files
Traceback (most recent call last):
  File "/workdir/env/bin/simtools-simulate-prod", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/workdir/external/simtools/simtools/applications/simulate_prod.py", line 185, in main
    pack_for_register(logger, simulator, args_dict)
  File "/workdir/external/simtools/simtools/applications/simulate_prod.py", line 150, in pack_for_register
    tar.add(file_to_tar, arcname=Path(file_to_tar).name)
  File "/usr/lib64/python3.11/tarfile.py", line 2171, in add
    tarinfo = self.gettarinfo(name, arcname)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/tarfile.py", line 2044, in gettarinfo
    statres = os.lstat(name)
              ^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pytest-of-root/pytest-0/popen-gw2/test-data10/simtools-simulate-prod-gamma_20_deg_pack_for_south_grid/simtools-output/simtel/logs/run000010_gamma_za20deg_azm000deg_South_test_layout_test-production-South-pack-for-grid.hdata.zst'
```

**Comment by GernotMaier** [comment]
If I just would know what `ctsim.hdata` is. Seems to be nowhere specified.

**Comment by tobiaskleiner** [comment]
From the sim_telarray documentation: 

```
HISTOGRAM_FILE
(type: Text, items: 0, default: "ctsim.hdata")
File with histograms (use ’hdata2hbook’ to convert to hbook or ’hdata2root’ to con- vert to ROOT-format histogram files).
```

**Comment by orelgueta** [comment]
It seems that when the test fails while using the `pack_for_grid` option it still copies whatever files it finds into the base simtools directory. I will open a separate issue to ignore the packing step if the first step failed.

**Comment by GernotMaier** [comment]
Weird thing is that there is no `ctsims.dat` when running 

```
pytest --no-cov -n auto tests/integration_tests/
```

There is one when adding the model version (but 6.0.0 should be the default...):

```
pytest --model_version="6.0.0" --no-cov -n auto tests/integration_tests/
```

This is the reason why I didn't notice it before.

**Comment by GernotMaier** [comment]
This required a bit of searching: the issue was with the simulate_light_emission tests, which did not have an explicit setting of `-C HISTOGRAM_FILE`. This generated the ctsim.hdata file.

**Comment by GernotMaier** [comment]
I think there are two bugs and the one you mention above is weird. The full output is:

```
INFO::simulate_prod(l193)::main::Production run is complete for primary gamma showers coming from 180.0 deg azimuth and zenith angle of 20.0 deg at the South site, using the 6.0.0 simulat     ion model.
2128 INFO::simulate_prod(l157)::pack_for_register::Packing the output files for registering on the grid
2129 INFO::simulator(l450)::get_file_list::Getting list of output files
2130 INFO::simulator(l450)::get_file_list::Getting list of log files
2131 INFO::simulator(l450)::get_file_list::Getting list of hist files
2132 Traceback (most recent call last):
2133   File "/workdir/env/bin/simtools-simulate-prod", line 8, in <module>
2134     sys.exit(main())
2135              ^^^^^^
2136   File "/workdir/external/simtools/simtools/applications/simulate_prod.py", line 201, in main
2137     pack_for_register(logger, simulator, args_dict)
2138   File "/workdir/external/simtools/simtools/applications/simulate_prod.py", line 166, in pack_for_register
2139     tar.add(file_to_tar, arcname=Path(file_to_tar).name)
2140   File "/usr/lib64/python3.11/tarfile.py", line 2171, in add
2141     tarinfo = self.gettarinfo(name, arcname)
2142               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2143   File "/usr/lib64/python3.11/tarfile.py", line 2044, in gettarinfo
2144     statres = os.lstat(name)
2145               ^^^^^^^^^^^^^^
2146 FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pytest-of-root/pytest-4/popen-gw1/test-data22/simtools-simulate-prod-gamma_20_deg_south_check_output/simtools-output/simtel/logs/run000001_gamma_za20deg_azm180deg_South_alpha_check_output.hdata.zst'
```

Somehow the code searches in the log directory for a data file.

This file exists where it should be: `/tmp/pytest-of-root/pytest-4/popen-gw1/test-data22/simtools-simulate-prod-gamma_20_deg_south_check_output/simtools-output/simtel/data/run000001/run000001_gamma_za20deg_azm180deg_South_alpha_check_output.zst`

**Comment by tobiaskleiner** [comment]
> I think there are two bugs and the one you mention above is weird. The full output is:
> 
> ```
> INFO::simulate_prod(l193)::main::Production run is complete for primary gamma showers coming from 180.0 deg azimuth and zenith angle of 20.0 deg at the South site, using the 6.0.0 simulat     ion model.
> 2128 INFO::simulate_prod(l157)::pack_for_register::Packing the output files for registering on the grid
> 2129 INFO::simulator(l450)::get_file_list::Getting list of output files
> 2130 INFO::simulator(l450)::get_file_list::Getting list of log files
> 2131 INFO::simulator(l450)::get_file_list::Getting list of hist files
> 2132 Traceback (most recent call last):
> 2133   File "/workdir/env/bin/simtools-simulate-prod", line 8, in <module>
> 2134     sys.exit(main())
> 2135              ^^^^^^
> 2136   File "/workdir/external/simtools/simtools/applications/simulate_prod.py", line 201, in main
> 2137     pack_for_register(logger, simulator, args_dict)
> 2138   File "/workdir/external/simtools/simtools/applications/simulate_prod.py", line 166, in pack_for_register
> 2139     tar.add(file_to_tar, arcname=Path(file_to_tar).name)
> 2140   File "/usr/lib64/python3.11/tarfile.py", line 2171, in add
> 2141     tarinfo = self.gettarinfo(name, arcname)
> 2142               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 2143   File "/usr/lib64/python3.11/tarfile.py", line 2044, in gettarinfo
> 2144     statres = os.lstat(name)
> 2145               ^^^^^^^^^^^^^^
> 2146 FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pytest-of-root/pytest-4/popen-gw1/test-data22/simtools-simulate-prod-gamma_20_deg_south_check_output/simtools-output/simtel/logs/run000001_gamma_za20deg_azm180deg_South_alpha_check_output.hdata.zst'
> ```
> 
> Somehow the code searches in the log directory for a data file.
> 
> This file exists where it should be: `/tmp/pytest-of-root/pytest-4/popen-gw1/test-data22/simtools-simulate-prod-gamma_20_deg_south_check_output/simtools-output/simtel/data/run000001/run000001_gamma_za20deg_azm180deg_South_alpha_check_output.zst`

Yes, can confirm that the file is in `simtel/data/run000001/` directory and the file is searched wrongfully in the log directory.

**Comment by orelgueta** [comment]
> I think there are two bugs and the one you mention above is weird. The full output is:
> 
> ```
> INFO::simulate_prod(l193)::main::Production run is complete for primary gamma showers coming from 180.0 deg azimuth and zenith angle of 20.0 deg at the South site, using the 6.0.0 simulat     ion model.
> 2128 INFO::simulate_prod(l157)::pack_for_register::Packing the output files for registering on the grid
> 2129 INFO::simulator(l450)::get_file_list::Getting list of output files
> 2130 INFO::simulator(l450)::get_file_list::Getting list of log files
> 2131 INFO::simulator(l450)::get_file_list::Getting list of hist files
> 2132 Traceback (most recent call last):
> 2133   File "/workdir/env/bin/simtools-simulate-prod", line 8, in <module>
> 2134     sys.exit(main())
> 2135              ^^^^^^
> 2136   File "/workdir/external/simtools/simtools/applications/simulate_prod.py", line 201, in main
> 2137     pack_for_register(logger, simulator, args_dict)
> 2138   File "/workdir/external/simtools/simtools/applications/simulate_prod.py", line 166, in pack_for_register
> 2139     tar.add(file_to_tar, arcname=Path(file_to_tar).name)
> 2140   File "/usr/lib64/python3.11/tarfile.py", line 2171, in add
> 2141     tarinfo = self.gettarinfo(name, arcname)
> 2142               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 2143   File "/usr/lib64/python3.11/tarfile.py", line 2044, in gettarinfo
> 2144     statres = os.lstat(name)
> 2145               ^^^^^^^^^^^^^^
> 2146 FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pytest-of-root/pytest-4/popen-gw1/test-data22/simtools-simulate-prod-gamma_20_deg_south_check_output/simtools-output/simtel/logs/run000001_gamma_za20deg_azm180deg_South_alpha_check_output.hdata.zst'
> ```
> 
> Somehow the code searches in the log directory for a data file.
> 
> This file exists where it should be: `/tmp/pytest-of-root/pytest-4/popen-gw1/test-data22/simtools-simulate-prod-gamma_20_deg_south_check_output/simtools-output/simtel/data/run000001/run000001_gamma_za20deg_azm180deg_South_alpha_check_output.zst`

Wait, this is not a data file, this is the histogram file which is technically part of the log, so it is in the right location.

**Comment by GernotMaier** [comment]
Good to know. Then the issue is that it this file is written into the data and not into the log directory.

**Comment by orelgueta** [comment]
I cannot reproduce the error when using this branch. Can you please provide detailed instructions on how to reproduce it?

**Comment by GernotMaier** [comment]
This is what you do to reproduce the issue (it is hidden, this is why we did not notice it earlier):

- replace line 65 in tests/integration_tests/test_applications_from_config.py with `config_files = sorted(Path(__file__).parent.glob("config/*gamma_20_deg_*_check_output.yml"))` - this will run the integration tests using only those files (I am pretty sure there is another trick to run only a subset of the tests)
- run the integration tests with `pytest --no-cov -n auto --count 5 --random-order tests/integration_tests/test_applications_from_config.py`

**Comment by GernotMaier** [comment]
This might be an effect due to

- running jobs in parallel
- jobs writing histogram files with the same name into the simtools directory (where we start tests)
- the cleanup of one job removes the files from another one

I do think we should never write into the directory we are starting a job from (see also issue #1167 for a similar issue for the illuminator simulations)

**Comment by orelgueta** [comment]
> This is what you do to reproduce the issue (it is hidden, this is why we did not notice it earlier):
> 
> * replace line 65 in tests/integration_tests/test_applications_from_config.py with `config_files = sorted(Path(__file__).parent.glob("config/*gamma_20_deg_*_check_output.yml"))` - this will run the integration tests using only those files (I am pretty sure there is another trick to run only a subset of the tests)

You can do this instead by just running the command below:

`pytest -n auto --count 5 --random-order --no-cov tests/integration_tests/ -k simtools-simulate-prod`

It will run also the the other zenith angles, but that's not an issue.

**Comment by orelgueta** [comment]
I am pretty sure the issue comes from the `directory_for_grid_upload` option which creates a tarball for the grid which by definition is in a predefined, easy to find, directory. When running in parallel, somehow one job overwrites a different one and also the output path is not defined properly I think. I can take a look, but my schedule this week is a bit weird, so it might take a while.

**Comment by GernotMaier** [comment]
I am not sure - each of those tests is running in its own directory.

When comparing successful and failed log files from sim_telarray, I am noticing that those failed have no log file entries from running sim_telarray, see attached screenshot.

Obviously no histogram files if sim_telarray is not running. Question is just what is going on. I still suspect that they are using a common tmp directory defined by sim_telarray and one is overwriting the other.

Screenshot 2024-10-15 at 16 39 32

**Comment by orelgueta** [comment]
OK, I indeed see something strange which also affects the main. Do you want to wait for me to look into it or will you?

**Comment by GernotMaier** [comment]
I am trying to look into it, but it is hard.

@tobiaskleiner - could you still review this PR? This problem is independent of this PR (I can reproduce the problem on main, so this is not an issue of this PR).

File: tests/integration_tests/config/validate_cumulative_psf_prod6_psf.yml [hunk 1]
```diff
@@ -8,6 +8,6 @@ CTA_SIMPIPE:
     DATA: ./tests/resources/PSFcurve_data_v2.txt
     ZENITH: 20
     TEST: true
-    OUTPUT_PATH: simtools-tests
+    OUTPUT_PATH: simtools-output
   INTEGRATION_TESTS:
-    - OUTPUT_FILE: ray-tracing/results/ray-tracing-North-LSTN-01-d10.0km-za20.0deg_validate_cumulative_psf.ecsv
+    - OUTPUT_FILE: results/ray-tracing-North-LSTN-01-d10.0km-za20.0deg_validate_cumulative_psf.ecsv
```
Comment (tobiaskleiner) [inline tests/integration_tests/config/validate_cumulative_psf_prod6_psf.yml:13]: Just want to make sure `results` is intended here in the output file name? ray-tracing would be `sub_dir`. Is this correct? `results` appears also in several other places.

**Review by tobiaskleiner — COMMENTED** [review]
Thanks @GernotMaier. Added one comment. Otherwise all good and we can move the unrelated failing of the integratration tests to another issue.

File: tests/integration_tests/config/validate_cumulative_psf_prod6_psf.yml (hunk 1 above)
Comment (GernotMaier) [reply tests/integration_tests/config/validate_cumulative_psf_prod6_psf.yml:13]: Yes, this was changed to make it consistent with the other tests.

**Comment by GernotMaier** [comment]
Thanks @tobiaskleiner !

**Comment by GernotMaier** [comment]
Don't forget to approve by in the review field....

**Review by tobiaskleiner — APPROVED** [review]

**Pull request merged**